### DISCUSSION TITLE
Insert a string into a json without parsing it

### DISCUSSION THREAD

**Discussion by tawmoto** [nlohmann/json#2904] [Q&A] [answered]
Hi,

I have a `std::string` containing a huge valid json (50.000 elements) and I want to "plug" it into another `json` but without parsing it.

Now I am doing smth like this:
`jsonObject["field"] = str.parse();`
but it's very slow (20+ seconds) and I really don't need to parse it to read the data, just "plug" it into the `jsonObject`.

Can be done in any way?

Thank you

**Comment by nlohmann** [comment, marked as answer]
Unfortunately not with this library as it is DOM based.

**Reply by tawmoto** [reply]
I see, thank you for your answe, Niels.
